### PR TITLE
Add annotations to namespace collector

### PR DESCRIPTION
**Pull request by pickledrick**

### CODE REVIEW

File: Documentation/namespace-metrics.md [hunk 1]
```diff
@@ -4,4 +4,5 @@
 | ---------- | ----------- | ----------- |
 | kube_namespace_status_phase| Gauge | `namespace`=&lt;ns1&gt; <br> `status`=&lt;Active\|Terminating&gt; |
 | kube_namespace_labels | Gauge | `namespace`=&lt;ns1&gt; <br> `label_NS_LABEL`=&lt;NS_LABEL&gt; |
+| kube_namespace_annotations | Gauge | `namespace`=&lt;ns1&gt; <br> `label_NS_LABEL`=&lt;NS_LABEL&gt; |
```
Comment (andyxning) [inline Documentation/namespace-metrics.md]: `s/_LABEL/_NAMESPACE`

**Comment by andyxning** [comment]
@pickledrick Just one nit. PTAL. Others lgtm.

File: Documentation/namespace-metrics.md (hunk 1 above)
Comment (andyxning) [reply Documentation/namespace-metrics.md]: BTW, maybe we should add a `kubeAnnotationsToPrometheusLabels` and make the prefix for  annotations being `annotation_`. WDYT @brancz

Comment (pickledrick) [reply Documentation/namespace-metrics.md]: yeh wasn't sure if a new function made sense, but happy whatever the decision @andyxning

File: Documentation/namespace-metrics.md [hunk 2]
```diff
@@ -4,4 +4,5 @@
 | ---------- | ----------- | ----------- |
 | kube_namespace_status_phase| Gauge | `namespace`=&lt;ns1&gt; <br> `status`=&lt;Active\|Terminating&gt; |
 | kube_namespace_labels | Gauge | `namespace`=&lt;ns1&gt; <br> `label_NS_LABEL`=&lt;NS_LABEL&gt; |
+| kube_namespace_annotations | Gauge | `namespace`=&lt;ns1&gt; <br> `label_NS_ANNOTATION`=&lt;NS_ANNOTATIONL&gt; |
```
Comment (andyxning) [inline Documentation/namespace-metrics.md]: `s/label_NS_ANNOTATION/annotation_NS_ANNOTATION`

Comment (andyxning) [reply Documentation/namespace-metrics.md]: @pickledrick Just last one nit. PTAL.

Comment (pickledrick) [reply Documentation/namespace-metrics.md]: 👍 Have fixed up @andyxning

**Comment by andyxning** [comment]
Thanks @pickledrick 

/lgtm
/approve

**Pull request merged**